### PR TITLE
Better handling end errors

### PR DESCRIPTION
**Pull request by Bod9001**
puts more Try catches on end game

### Changelog:

CL: [Fix] fix is an error that could happen with RoundEndScoreBuilder.cs

### CODE REVIEW

File: UnityProject/Assets/Scripts/Objects/Nuke.cs [hunk 1]
```diff
@@ -148,6 +148,7 @@ private void Detonate()
 			}
 			else
 			{
+				GameManager.Instance.RoundEndTime = 15;
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Objects/Nuke.cs]: uhm? What's this?

Comment (Bod9001) [reply UnityProject/Assets/Scripts/Objects/Nuke.cs]: Makes the round time not too long for nuke ending

Comment (corp-0) [reply UnityProject/Assets/Scripts/Objects/Nuke.cs]: normally what's the ending time?

Comment (Bod9001) [reply UnityProject/Assets/Scripts/Objects/Nuke.cs]: but is not needed technically

Comment (corp-0) [reply UnityProject/Assets/Scripts/Objects/Nuke.cs]: Yeah, unless you want people to chat a while before starting the next one

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Objects/Nuke.cs]: doesn't the video hide the chat UI?

Comment (Bod9001) [reply UnityProject/Assets/Scripts/Objects/Nuke.cs]: shall remove it

Comment (corp-0) [reply UnityProject/Assets/Scripts/Objects/Nuke.cs]: ye

**Pull request merged**